docs/output.md

```diff
@@ -442,7 +444,54 @@ Files created:
 
 </details>
 
-#### Sentieon Haplotyper
```

Comment:
Has this section title moved somehwere else or just disappeared?

Reply:
Messed it up. How about now? Better?